plugins/qsi-dock/README.md

```diff
@@ -0,0 +1,32 @@
+# QSI Dock
+
+## Introduction
+
+This plugin uses the MCU to write all the dock firmware components. The MCU version
+is provided by the DMC bcdDevice.
```

Comment:
MCU and DMC should probably be explained

Reply:
Done.